changelog/23690.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
@robmonte next time please use the correct new feature formatting for new features in the changelog.

Reply:
To add some more clarification - this should use the "Feature Name" we'd use in the release notes, and should try to communicate to the user why they would want to learn more about this brand new feature we're introducing in Vault 1.16.